src/models/converters/transactions/mod.rs

```diff
@@ -28,6 +25,21 @@ impl MultisigTransaction {
         self.confirmations_required.unwrap_or(threshold)
     }
 
+    fn missing_signers(&self, owners: &Vec<String>) -> Vec<String> {
+        match &self.confirmations {
+            Some(confirmations) => {
+                let mut missing_signers = vec![];
+                for owner in owners {
+                    if !confirmations.iter().any(|c| &c.owner == owner) {
+                        missing_signers.push(owner.to_owned());
+                    }
+                }
+                missing_signers
+            },
+            None => owners.clone(),
+        }
```

Comment:
Your proposed code does something completely different than my code

Reply:
but yeah I can use `filter_map` for sure ... for `map_or_else` the default value is always instanciated, which here would mean it would always clone the owners ... not really optimal

Reply:
well yeah I wrote that as a suggestion only, maybe you need to flip the condition of the `if` I am not sure it is "completely" different TBH. Couldn't you use `owners.to_owned()` as the default?

Reply:
`to_owned` peforms `clone` in most cases (and here too)

Reply:
`map_or_else` takes a function for the default value, so in that case it works ... adjusted the code